Cargo.toml

```diff
@@ -26,7 +26,7 @@ aws-lc-rs = ["rustls/aws-lc-rs", "webpki/aws_lc_rs"]
 
 [dependencies]
 # Keep in sync with RUSTLS_CRATE_VERSION in build.rs
-rustls = { version = "0.23.13", default-features = false, features = ["std", "tls12"] }
+rustls = { version = "0.23.14", default-features = false, features = ["std", "tls12"] }
```

Comment:
0.23.18?

Reply:
Argh how did I manage to get the version right everywhere but where it mattered the most :facepalm: Thanks for catching.